### ISSUE TITLE
svelte: error path package.json

### ISSUE THREAD

**Issue by gawlk** [FormidableLabs/urql#770]
Hi,

I'm having a hard time importing the library. I'm not sure what the problem is, since it seems to work correctly for others.

Here's the error:

```
~/code/svelte/my-svelte-project
➜ yarn dev
yarn run v1.22.4
warning package.json: No license field
$ rollup -c -w
rollup v1.32.1
bundles src/main.js → public/build/bundle.js...
[!] (plugin svelte) Error: Package subpath './package.json' is not defined by "exports" in /Users/k/code/svelte/my-svelte-project/node_modules/@urql/svelte/package.json
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in /Users/k/code/svelte/my-svelte-project/node_modules/@urql/svelte/package.json
    at applyExports (internal/modules/cjs/loader.js:492:9)
    at resolveExports (internal/modules/cjs/loader.js:508:23)
    at Function.Module._findPath (internal/modules/cjs/loader.js:632:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1001:27)
    at Function.requireRelative.resolve (/Users/k/code/svelte/my-svelte-project/node_modules/require-relative/index.js:30:17)
    at tryResolve (/Users/k/code/svelte/my-svelte-project/node_modules/rollup-plugin-svelte/index.js:50:19)
    at Object.resolveId (/Users/k/code/svelte/my-svelte-project/node_modules/rollup-plugin-svelte/index.js:177:21)
    at /Users/k/code/svelte/my-svelte-project/node_modules/rollup/dist/shared/node-entry.js:13117:25
```
 
I'm using @urql/svelte: 0.2.2

**Steps to reproduce**

```
npx degit sveltejs/template my-svelte-project
cd my-svelte-project
yarn add @urql/svelte graphql
# Add 'import { initClient } from "@urql/svelte";' in src/App.svelte
yarn dev
```

I'm using node 14 not sure if it's causing issues ? Other than that my setup is pretty normal.

**Expected behavior**

A successful build

**Actual behavior**

Cannot build

EDIT: Actually if I add `"./package.json": "./package.json",` in the `exports` in `package.json`, will complain about the same thing but the core package and if I add it there, same thing in the wonka package. After that it seems to be working, at least it builds.

**Comment by kitten** [comment]
This should be fixed in `@urql/svelte@0.2.3` and the various other patch bumps that have just been released.

**Comment by gawlk** [comment]
Great, thanks !
